### DISCUSSION TITLE
How to get unrounded corners with Mask-RCNN

### DISCUSSION THREAD

**Discussion by Yink** [facebookresearch/detectron2#4918] [Q&A]
Hi there,

I have a training set of rectangular fabric objects that I need the exact dimensions of. The training data is labelled precisely, but for some reason the predictions always round the corners a bit. Do you have any idea how to improve precision of the mask generation?
